api/resources/translations/messages-es.properties

```
@@ -598,241 +598,306 @@ debug.db_info.name = Nombre
+login.token.general.error = Algo salió mal al procesar su solicitud. Inténtalo de nuevo. Si esto vuelve a suceder, comuníquese con el administrador para recibir un nuevo enlace.
+login.token.invalid = Su enlace no es válido. Póngase en contacto con el administrador para recibir un nuevo enlace.
+login.token.loading = Iniciando sesión. Espere por favor.
+login.token.missing = A su enlace le falta información requerida. Póngase en contacto con el administrador para recibir un nuevo enlace.
+login.token.redirect.login = Página de inicio de sesión
+login.token.redirect.login.info = Si conoce su nombre de usuario y contraseña, haga clic en el siguiente enlace para cargar la página de inicio de sesión.
+login.token.timeout = Se agotó el tiempo de espera al intentar iniciar sesión. Inténtelo de nuevo. Si esto vuelve a suceder, comuníquese con el administrador para recibir un nuevo enlace.
+login.unsupported_browser = Para una mejor experiencia con la aplicación, comuníquese con su administrador o supervisor.
+login.unsupported_browser.outdated_cht_android = Hágales saber que es necesario actualizar la versión del cht-android.
+login.unsupported_browser.outdated_webview_apk = Hágales saber que es necesario actualizar la versión del Webview APK.
+login.unsupported_browser.outdated_browser = Hágales saber que es necesario actualizar su navegador.
+message.characters.left = Quedan {{characters}} carácteres
+message.characters.left.multiple = {{messages}} SMS, quedan {{characters}} carácteres
+message.characters.left.multiple.many = {{messages}} SMS, quedan {{characters}} carácteres - \¡demasiados mensajes\!
+message.list.complete = No más mensajes
+messages.c.report_accepted = Gracias {{contact.name}} por registrar a {{patient_name}}. Su ID es {{patient_id}}. Han sido inscritos en el programa de salud infantil.
+messages.c.validation.months_since_birth = El formato de registro para {{patient_id}} es incorrecto, asegúrese de que la edad esté en meses y entre 0 y 59.
+messages.c.validation.patient_name = {{\#patient_name}}El formato de registro es incorrecto. Comience con C seguida de un espacio, edad en semanas seguida de un espacio y el nombre del niño (máximo de 30 caracteres).{{/patient_name}}{{^patient_name}}. Falta el nombre en el formato de registro. Comience con C seguida de un espacio, la edad en semanas seguida de un espacio y el nombre del niño {{/patient_name}}.
+messages.c.validation.weeks_since_birth = El formato de registro para {{patient_id}} es incorrecto, asegúrese de que la edad esté en semanas y entre 0 y 260.
+messages.d.report_accepted = Gracias {{contact.name}} por informar el parto de {{patient_name}} ({{patient_id}}).
+messages.d.validation.days_since_birth = El informe de entrega de {{patient_id}} es incorrecto. Asegúrese de que los días transcurridos desde la entrega sean un número de días entre 0 y 365, o déjelo en blanco.
```

```suggestion
messages.d.validation.days_since_birth = El informe del parto del {{patient_id}} es incorrecto. Asegúrese de que los días transcurridos desde el parto sean un número de días entre 0 y 365, o déjelo en blanco.
```

Sorry, I missed this before.